gapis/shadertools/cc/libmanager.cpp

```diff
@@ -228,6 +228,15 @@ code_with_debug_info_t* convertGlsl(const char* input, size_t length, const opti
 
   std::string source = spirv2glsl(std::move(spirv_new));
 
+  // Add support for layout(location) as needed by separate shader objects
+  // TODO: Postprocessing source code like this is fairly hacky. Find a better way.
+  auto extensionPos = source.find("#ifdef GL_ARB_shading_language_420pack");
```

Comment:
This is a bit gross. Maybe we should use a regex to at least handle more than 1 whitespace between the #ifdef and the extension name?